src/tabs/tab-header-bar.tsx

```diff
@@ -332,12 +348,55 @@ const TabTrigger = forwardRef(
     ref: React.Ref<HTMLElement>
   ) => {
     const refObject = useRef<HTMLElement>(null);
+    const childRefObject = useRef<HTMLElement>(null);
```

Comment:
Would it make sense to rename the refs to e.g. `tabRefObject` and `tabLabelRefObject` for better semantic meaning?

Reply:
Why an extra ref is needed at all, can the refObject be used instead?